feature1.txt

```diff
@@ -0,0 +1 @@
+this file was created using feature1 branch
```

Comment:
This is the rigth way to use both accounts

Reply:
Thanks for participating